scss/partials/_filter-bar.scss

```diff
@@ -111,7 +111,7 @@
   &__filter-icon {
     width: 3rem;
     height: 100%;
-    mask: url("/img/filter.svg") no-repeat center / contain;
+    mask: url("data:image/svg+xml,%3Csvg width='32' height='32' viewBox='0 0 32 32' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cline x1='30' y1='10' x2='13' y2='10' stroke='white' stroke-width='2'/%3E%3Ccircle cx='10' cy='10' r='4' stroke='white' stroke-width='2'/%3E%3Cline x1='6' y1='10' x2='2' y2='10' stroke='white' stroke-width='2'/%3E%3Cline x1='2' y1='22' x2='19' y2='22' stroke='white' stroke-width='2'/%3E%3Ccircle cx='22' cy='22' r='4' transform='rotate(-180 22 22)' stroke='white' stroke-width='2'/%3E%3Cline x1='26' y1='22' x2='30' y2='22' stroke='white' stroke-width='2'/%3E%3C/svg%3E%0A") no-repeat center / contain;
```

Comment:
Yeah, I like this.

Reply:
It looks like `filter.svg` is not used anywhere else. Should we keep it or remove it in this case?

Reply:
You're right. Most of the image files in that folder to be removed I think. I'll remove the filter svg for now.